paddle/platform/gpu_info.h

```diff
@@ -18,10 +18,14 @@ limitations under the License. */
 
 #include <cuda_runtime.h>
 #include <stddef.h>
+#include <string>
 
 namespace paddle {
 namespace platform {
 
+//! Environment variable: fraction of GPU memory to use on each device.
+const std::string kEnvFractionGpuMemoryToUse = "FRACTION_GPU_MEMORY_TO_USE";
```

Comment:
Environment variable often have prefix to distinguash between applications, to prevent if two application use the same name. Maybe start with "PADDLE_"?